app-misc/calamares-gentoo-livecd/calamares-gentoo-livecd-3.3.14_p20250915-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="8"
+
+MY_PN="calamares"
+MY_P="${MY_PN}-${PV}"
+PYTHON_COMPAT=( python3_{11..13} )
+
+inherit desktop python-single-r1
+
+DESCRIPTION="Gentoo Linux ${MY_PN} installer configuration for LiveCD"
+HOMEPAGE="https://github.com/StefanCristian/calamares-gentoo-livecd"
+SRC_URI="https://github.com/StefanCristian/${PN}/archive/refs/tags/${PV}.tar.gz -> ${PF}.tar.gz
```

Comment:
We usually avoid `${PF}` here, use `${P}` instead.  If upstream tarball changes, do a version bump, not a revision bump; revision bumps should reuse the same tarball